DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs

```diff
@@ -65,7 +65,7 @@ public void Section_content_should_have_consolidation_path_when_courseSetting_ex
                 consolidationPath: consolidationPath,
                 courseSettings: null
             );
-            var expectedConsolidationPath = config.GetConsolidationPathUrl(consolidationPath);
+            var expectedConsolidationPath = $"{config["CurrentSystemBaseUrl"]}/tracking/dlconsolidation?client={consolidationPath}";
```

Comment:
This should use the BaseUrl variable. It shouldn't use the config at all. We want it to not rely on any external methods. Does that make sense? 🙂

Reply:
Ah okay yep that makes sense